packages/peerplays/lib/peerplays.js

```diff
@@ -306,19 +306,23 @@ export default class PPY extends Plugin {
     // Get the transaction
     let transferTransaction = await _PPY.getTransferTransaction(from, to, amount, memo, '1.3.0');
 
+    let privateActiveKey = await KeyPairService.publicToPrivate(account.publicKey);
```

Comment:
you're taking the account public key and calling it private key?

Reply:
I'm taking the public key and retrieving the associated key within the keypair needed to sign the transaction.